test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1

```diff
@@ -242,7 +242,7 @@ Describe 'Positive Parse Properties Tests' -Tags "CI" {
                 $this.h = [ordered] @{}
             }
         }
-        [A]::new().h.GetType().Name | Should Be 'OrderedDictionary'
+        [A]::new().h.GetType().Name | Should -Be 'OrderedDictionary'
```

Comment:
Search and replace any instances of `GetType()` on the left side to use `Should -BeOfType` on the right side.